helmclient.go

```diff
@@ -425,6 +413,49 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 	return nil
 }
 
+func (c *Client) ListReleaseContents() ([]*ReleaseContent, error) {
+	var resp *hapiservices.ListReleasesResponse
+	{
+		o := func() error {
+			t, err := c.newTunnel()
+			if IsTillerNotFound(err) {
+				return backoff.Permanent(microerror.Mask(err))
+			} else if err != nil {
+				return microerror.Mask(err)
+			}
+			defer c.closeTunnel(t)
+
+			resp, err = c.newHelmClientFromTunnel(t).ListReleases()
```

Comment:
I don't know the api but maybe it requires paging.

Reply:
Possibly. We have < 10 releases, so I'm not too worried about it.

Reply:
I am. I can recall problems with paging ELBs. Recently with paging VMSS instances..

```
// ListReleasesResponse is a list of releases.
type ListReleasesResponse struct {
	// Count is the expected total number of releases to be returned.
	Count int64 `protobuf:"varint,1,opt,name=count" json:"count,omitempty"`
	// Next is the name of the next release. If this is other than an empty
	// string, it means there are more results.
	Next string `protobuf:"bytes,2,opt,name=next" json:"next,omitempty"`
	// Total is the total number of queryable releases.
	Total int64 `protobuf:"varint,3,opt,name=total" json:"total,omitempty"`
	// Releases is the list of found release objects.
	Releases []*hapi_release5.Release `protobuf:"bytes,4,rep,name=releases" json:"releases,omitempty"`
}
```

Check `Next`.

Reply:
Then I shall worry about it

Reply:
Easy man, I can not be lazy heh

Reply:
Done